src/js/arrays_loops.js

```diff
@@ -0,0 +1,131 @@
+import { contacts } from "../data/contacts.js";
+
+console.log(contacts);
```

Comment:
remove unneeded consoles when handing in your work (exposes data,impacts performance)